packages/edit-site/src/components/editor/index.js

```diff
@@ -242,6 +243,7 @@ function Editor( { initialSettings } ) {
 										}
 										actions={
 											<>
+												<EditorSnackbars />
```

Comment:
One thing that you should test is whether it is possible to tab to the snackbar after you complete the action like publishing. In general, it's a challenge because the order of HTML elements matters here. I don't know how often we create snackbars as of today and how many of them have actionable items like links or inner buttons. Ideally, it's only a few tabs away when you want to use them.

Reply:
I tried but not able to tab to the snackbar in Gutenberg 11.0.